### PR TITLE
Add Environment Variables Example

### LINKED ISSUE #14570: Add environment variables example
We should have more examples about our core features, one of those is [Environment Variables](https://nextjs.org/docs/basic-features/environment-variables).

The example should be added to the examples folder and linked by our docs

### LINKED ISSUE #14576: Remove the with-now-env example
Removing means update to example to only contain the readme, and the readme should point to the alternatives.

### PR DESCRIPTION
**Pull request by lfades**
Closes https://github.com/vercel/next.js/issues/14570
Closes https://github.com/vercel/next.js/issues/14576

- The example features everything on the page in our docs while still being quite simple
- Added a link to it in the documentation
- Deprecated the with-now-env example

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 12s | 12.1s | ⚠️ +145ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.98 | 2.01 | ⚠️ +0.03 |
| / avg req/sec | 1262.62 | 1243.57 | -19.05 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.136 | 1.223 | ⚠️ +0.09 |
| /error-in-render avg req/sec | 2201.2 | 2043.51 | -157.69 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 946 B | 946 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 13s | 12.9s | -111ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: 68becb1cc776b6c07de179a21dd19d0890c3c4e5

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 11.5s | 11.6s | ⚠️ +144ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.928 | 1.903 | -0.02 |
| / avg req/sec | 1296.54 | 1313.47 | ⚠️ +16.93 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.165 | 1.119 | -0.05 |
| /error-in-render avg req/sec | 2145.61 | 2233.75 | ⚠️ +88.14 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| index.html gzip | 955 B | 955 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 946 B | 946 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.7s | ⚠️ +512ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: 731d623d254a5cedead2c670e4cf4be0c84a1c52

**Review by timneutkens — APPROVED** [review]

File: examples/environment-variables/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import styles from '../styles.module.css'
+
+// The value here will be defined in the terminal, because of server side rendering,
+// but not in the browser console
+console.log('[Server only] ENV_VARIABLE:', process.env.ENV_VARIABLE)
```
Comment (timneutkens) [inline examples/environment-variables/pages/index.js:5]: This is generally not how we should show this example. Instead it should rely on `getStaticProps` / `getServerSideProps`

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 12.9s | 12.9s | ⚠️ +64ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.202 | 2.204 | 0 |
| / avg req/sec | 1135.31 | 1134.4 | -0.91 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.46 | 1.533 | ⚠️ +0.07 |
| /error-in-render avg req/sec | 1712.47 | 1630.28 | -82.19 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 13.6s | 13.9s | ⚠️ +291ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: 9d3332dcdbc7f92cbd9d811f028962407fed4360

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 11.4s | 11.4s | ⚠️ +61ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.948 | 1.909 | -0.04 |
| / avg req/sec | 1283.45 | 1309.67 | ⚠️ +26.22 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.35 | 1.357 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1851.7 | 1841.98 | -9.72 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.3s | ⚠️ +163ms |
| nodeModulesSize | 67.1 MB | 67.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..558d.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.1 kB | 57.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.59 kB | 5.59 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.08 kB | 7.08 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.6 kB | 52.6 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | lfades/next.js examples/environment-variables | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 876 kB | 876 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 908 kB | 908 kB | ✓ |
| withRouter.js | 908 kB | 908 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: b647cf18437b4046ba4eebb176126bebdcaaf36a

**Pull request merged**